mlir/lib/Dialect/XeGPU/IR/XeGPUDialect.cpp

```diff
@@ -30,6 +31,61 @@ void XeGPUDialect::initialize() {
       >();
 }
 
+bool XeGPUDialect::isEvenlyDistributable(llvm::ArrayRef<int64_t> shape,
```

Comment:
let's have the comment here as it was earlier

Reply:
fixed